src/Polly.Core/Retry/RetryConstants.cs

```diff
@@ -0,0 +1,24 @@
+namespace Polly.Retry;
+
+internal static class RetryConstants
+{
+    public const string StrategyType = "Retry";
+
+    public const string OnRetryEvent = "OnRetry";
+
+    public const RetryBackoffType DefaultBackoffType = RetryBackoffType.Exponential;
+
+    public const int DefaultRetryCount = 3;
+
+    /// <summary>
+    /// Maximal allowed retry counts unless infinite.
+    /// </summary>
+    public const int MaxRetryCount = 100;
+
+    /// <summary>
+    /// Maximal allowed BaseDelay (1 day).
+    /// </summary>
+    public const int MaxBaseDelay = 24 * 3600 * 1000;
```

Comment:
Let's add the unit into the name of the constant (`MaxBaseDelayMilliseconds`?)

Alternatively as `DefaultBaseDelay` is a `TimeSpan`, why not also make this a `TimeSpan` and then it can just be `TimeSpan.FromDays(1)`?

Reply:
Dropped this and move the ranges inline to TimeSpanAttribute:

![image](https://user-images.githubusercontent.com/103487740/230347619-0f2edb9f-202e-4236-9ef2-65744b6ebfbf.png)